packages/mrm-task-lintstaged/index.js

```diff
@@ -32,9 +33,21 @@ function task(config) {
 
 		// Prettier
 		if (pkg.get('devDependencies.prettier') && !pkg.get('devDependencies.eslint-plugin-prettier')) {
+			const prettierVersion = semver.coerce(pkg.get('devDependencies.prettier'));
 			const script = pkg.getScript('format');
 			const exts = getExtsFromCommand(script);
-			newRules.push([extsToGlob(exts, prettierExtensions), 'prettier --write']);
+			let suggestedExtensions = prettierExtensions;
```

Comment:
We shouldn't overwrite user config value.

Reply:
I thought `exts` is user extensions. And the second argument is default value. Tests also passing for user extensions.

Reply:
`exsts` are extensions defined in the `package.json` and `prettierExtensions` is from a config which has a default value.